cfviz

```diff
@@ -20,6 +17,23 @@ def main():
 
     render(graph)
 
+def open_cfn(argv):
+    if argv[1:2] != []:
+        input_filename = argv[1]
+        with open(input_filename) as h:
+            if (".yml" in input_filename):
```

Comment:
```suggestion
            if any(extension in input_filename for extension in ['.yml', '.yaml']):
```

Reply:
If you're interested in this then I'll get it merged. (I didn't merge it before because I think @renuez' suggestion above is probably a better solution.

Then please feel free to raise another PR with your suggested addition. Then we don't have to wait for @ennslen, who has probably lost interest in this by now.